File: MTWDM iOS Xamarin/App4/App4/ViewController.cs
using System;

using UIKit;

namespace App4
{
    public partial class ViewController : UIViewController
    {
        protected ViewController(IntPtr handle) : base(handle)
        {
            // Note: this .ctor should not contain any initialization logic.
        }

        partial void onChangeValue(UIKit.UISlider sender){

            lblSaludo.TextColor = UIColor.FromHSBA(hue: 0, saturation: 0, brightness: 0, alpha: sender.Value);
            Console.WriteLine(sender.Value);

            //lblSaludo.textColor = UIColor(hue: 0, saturation: 0, brightness: 0, alpha: CGFloat(sender.value))
            //print("\(sender.value)")
        }

        public override void ViewDidLoad()
        {
            base.ViewDidLoad();
            // Perform any additional setup after loading the view, typically from a nib.
            lblSaludo.TextColor = UIColor.FromHSBA(hue: 0, saturation: 0, brightness: 0, alpha: sliderAlfa.Value);
        }

        public override void DidReceiveMemoryWarning()
        {
            base.DidReceiveMemoryWarning();
            // Release any cached data, images, etc that aren't in use.
        }
    }
}

File: MTWDM iOS Swift /app4/app4/ViewController.swift
//
//  ViewController.swift
//  app4
//
//  Created by Felipe Hernandez on 21/09/18.
//  Copyright © 2018 Felipe Hernandez. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var lblSaludo: UILabel!
    
    @IBOutlet weak var sliderAlfa: UISlider!
    
    @IBAction func onChangeValue(_ sender: UISlider) {
        
        let slider = sender as! UISlider
        
        lblSaludo.textColor = UIColor(hue: 0, saturation: 0, brightness: 0, alpha: CGFloat(sender.value))
        
        
        print("\(slider.value)")
        
    }
    override func viewDidLoad() {
        super.viewDidLoad()
        
         lblSaludo.textColor = UIColor(hue: 0, saturation: 0, brightness: 0, alpha: CGFloat(sliderAlfa.value))
    }


}

